pkg/debian/build.sh

```diff
@@ -38,7 +37,7 @@ Version: ${APP_LONG_VERSION}
 Architecture: ${OS_ARCH}
 Section: database
 Priority: optional
-Depends: python3, libpq5 (>= 11.0), libgssapi-krb5-2
+Depends: ${PYTHON_BINARY}, libpq5 (>= 11.0), libgssapi-krb5-2
```

Comment:
How safe is this, considering PYTHON_BINARY is being set in a child function?

Reply:
We have set the PYTHON_BINARY  in the '**_setup_env**' function where other variables are set like {SERVERROOT}, {DISTROOT}, {APP_NAME} etc. I thought if those are being set, so PYTHON_BINARY should also be set.

Reply:
Good point. No problem then.